test/tests/auxkernels/vector_variable_nodal/tests

```diff
@@ -0,0 +1,9 @@
+[Tests]
+  issues = '#16076'
```

Comment:
We'll need a `design` parameter here that references a markdown file. We'll need a `VectorVariableMagnitudeAux.md` file in order to pass documentation testing. You can look at other `.md` files in the `framework/doc/content/source/auxkernels` directory for some examples. Looks like `MaterialRealAux.md` is a fine example. Once you've created `VectorVariableMagnitudeAux.md`, then you can put here
```
issues = '#16076'
design = 'VectorVariableMagnitudeAux.md'
```

Reply:
aight, working on that now